lib/index.js

```diff
@@ -73,6 +99,26 @@ function createNumberConversion(bitLength, typeOpts) {
     }
 }
 
+conversions["any"] = function (V) {
```

Comment:
If you trace through the logic here, this function is always a no-op (just, an expensive one). Because we don't distinguish between "Web IDL types" and "JS types" (since this is, after all, written in JS) none of the conversions here are valuable. You can just have the function be `return V`.